sunspot_solr/spec/server_spec.rb

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
is this extra line necessary? if you are using VSCode, [this](https://github.com/mdx-dev/platform-survival-guide/blob/master/editor-settings.md) is a good place to start for setting up your linting rules